### PR TITLE
python311Packages.betterproto: disable failing pydantic tests

### PR DESCRIPTION
**Pull request by fabaff**
## Description of changes
Fix build (https://hydra.nixos.org/build/246526205)

See https://github.com/NixOS/nixpkgs/pull/269633

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 281994` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.github</li>
    <li>python311Packages.aiogithubapi</li>
    <li>python311Packages.aiogithubapi.dist</li>
    <li>python311Packages.betterproto</li>
    <li>python311Packages.betterproto.dist</li>
    <li>python311Packages.sigstore</li>
    <li>python311Packages.sigstore-protobuf-specs</li>
    <li>python311Packages.sigstore-protobuf-specs.dist</li>
    <li>python311Packages.sigstore.dist</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 281994` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python311Packages.aiogithubapi</li>
    <li>python311Packages.aiogithubapi.dist</li>
  </ul>
</details>
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python311Packages.betterproto</li>
    <li>python311Packages.betterproto.dist</li>
    <li>python311Packages.sigstore</li>
    <li>python311Packages.sigstore-protobuf-specs</li>
    <li>python311Packages.sigstore-protobuf-specs.dist</li>
    <li>python311Packages.sigstore.dist</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 281994` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python311Packages.aiogithubapi</li>
    <li>python311Packages.aiogithubapi.dist</li>
    <li>python311Packages.betterproto</li>
    <li>python311Packages.betterproto.dist</li>
    <li>python311Packages.sigstore</li>
    <li>python311Packages.sigstore-protobuf-specs</li>
    <li>python311Packages.sigstore-protobuf-specs.dist</li>
    <li>python311Packages.sigstore.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/betterproto/default.nix [hunk 1]
```diff
@@ -63,13 +70,14 @@ buildPythonPackage rec {
   '';
 
   meta = with lib; {
-    description = "Clean, modern, Python 3.6+ code generator & library for Protobuf 3 and async gRPC";
+    description = "Code generator & library for Protobuf 3 and async gRPC";
```
Comment (nikstur) [inline pkgs/development/python-modules/betterproto/default.nix:82]: This is the upstream description. Not sure why we would want to change that.

**Review by nikstur — COMMENTED** [review]
Changes look fine but why did you bundle all of those in one PR?

**Comment by fabaff** [comment]
> Changes look fine but why did you bundle all of those in one PR?

All those builds are failing at the moment and are connected.

File: pkgs/development/python-modules/betterproto/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/betterproto/default.nix:82]: The "marketing part" or "how upstream thinks the project looks like" usually doesn't really add a lot of value to the description. If the library is modern and clean then we would not need to disable tests :wink: 

Sure, this change is highly opinionated.

**Pull request merged**